stock_report_deliveryslip_adj/report/report_deliveryslip.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
+        <xpath expr="//div[@class='page']" position="before">
+            <div class="row">
+                <div class="col-12 text-center my-4">
+                    <h2>Delivery Slip</h2>
+                </div>
+            </div>
+        </xpath>
+        <xpath
+            expr='//table[@t-if="o.move_line_ids and o.state==&apos;done&apos;"]/thead/tr/th'
+            position="after"
+        >
+            <th>
+                <strong>Quantity</strong>
+            </th>
+        </xpath>
+        <xpath expr="//span[@t-field='move_line.product_id']/.." position="after">
+            <td>
+                <t t-if="move_line.secondary_uom_qty">
+                    <span t-field="move_line.secondary_uom_qty" />
+                    <span t-field="move_line.secondary_uom_id.name" />
+                </t>
+                <t t-else="">
+                    <span t-field="move_line.qty_done" />
+                    <span t-field="move_line.product_uom_id" />
+                </t>
+            </td>
+        </xpath>
+        <xpath
+            expr='//table[@t-if="o.move_line_ids and o.state==&apos;done&apos;"]/thead/tr/th'
+            position="after"
+        >
+            <th>
+                <strong>Client Order Ref</strong>
+            </th>
+        </xpath>
+        <xpath expr="//span[@t-field='move_line.product_id']/.." position="after">
+            <td>
+                <span t-field="move_line.sale_line.client_order_ref"/>
+            </td>
+        </xpath>
+        <xpath expr="//span[@t-field='move_line.sale_price_unit']" position="attributes">
+            <attribute name="t-if">not move_line.sale_line.secondary_uom_price</attribute>
+        </xpath>
+        <xpath expr="//span[@t-field='move_line.sale_price_unit']" position="after">
+            <t t-if="move_line.sale_line.secondary_uom_price">
+                <span t-field="move_line.sale_line.secondary_uom_price" />
+                <span>/</span>
+                <span t-field="move_line.sale_line.secondary_uom_id.name" />
+            </t>
+        </xpath>
+        <xpath expr="//th[@class='text-center']" position="attributes">
+            <attribute name="style">display:none;</attribute>
+        </xpath>
+        <xpath expr="//td[@class='text-center']" position="attributes">
+            <attribute name="style">display:none;</attribute>
+        </xpath>
```

Comment:
It's impossible to tell what this adjustment is about without looking at the standard template.  Please comment which part this change intends to adjust.